modules/storages/app/services/storages/nextcloud_group_folder_properties_sync_service.rb

```diff
@@ -30,48 +30,60 @@
 
 module Storages
   class NextcloudGroupFolderPropertiesSyncService
-    using Peripherals::ServiceResultRefinements
+    extend ActiveModel::Naming
+    extend ActiveModel::Translation
 
-    PERMISSIONS_MAP = {
-      read_files: 1,
-      write_files: 2,
-      create_files: 4,
-      delete_files: 8,
-      share_files: 16
-    }.freeze
+    using Peripherals::ServiceResultRefinements
 
+    PERMISSIONS_MAP = { read_files: 1, write_files: 2, create_files: 4, delete_files: 8, share_files: 16 }.freeze
     PERMISSIONS_KEYS = PERMISSIONS_MAP.keys.freeze
     ALL_PERMISSIONS = PERMISSIONS_MAP.values.sum
     NO_PERMISSIONS = 0
 
+    include Injector["nextcloud.commands.create_folder", "nextcloud.commands.rename_file", "nextcloud.commands.set_permissions",
+                     "nextcloud.queries.group_users", "nextcloud.queries.file_ids", "nextcloud.authentication.userless",
+                     "nextcloud.commands.add_user_to_group", "nextcloud.commands.remove_user_from_group"]
```

Comment:
❓ How does it actually work? I see below methods like `remove_user_from_group` and `userless`. Are those the one included by the injector?

Reply:
I'm toying with this to avoid multiple calls to the Registry.

`AutoInject` basically creates a bunch of instance variables (and getters) for each command, using the least component of the container path. We can also rename the variables/getters if needed.

Not sure if it helps a lot, but wanted to give it a try. =)